app.js

```diff
@@ -359,10 +446,18 @@ async function openBrowser() {
 async function getNewStreamers(page) {
   header("Streamer Refresh");
   await page.goto(STREAMERS_URL, { waitUntil: "networkidle0" });
+  
+  //was the category found?
+  const notFound = await query(page, CATEGORY_NOT_FOUND);
 
+  if (notFound.length || notFound.text() == "Category does not exist") {
+     error(`Game category not found, did you enter the game as displayed on twitch?`);
+     shutdown();
+  }
+  
   debug("Checking login ...");
   await checkLogin(page);
-
+  
```

Comment:
Run `prettier` on the code to remove whitespace issues like these

Reply:
yeah I directly edited it on github.